src/goto-programs/goto_trace.h

```diff
@@ -243,18 +246,22 @@ void trace_value(
 
 #define OPT_GOTO_TRACE "(trace-json-extended)" \
                        "(trace-show-function-calls)" \
+                       "(trace-show-code)" \
                        "(trace-hex)"
 
 #define HELP_GOTO_TRACE                                                        \
   " --trace-json-extended        add rawLhs property to trace\n"               \
   " --trace-show-function-calls  show function calls in plain trace\n"         \
+  " --trace-show-code            show original code in plain trace\n"          \
```

Comment:
Description suggests we're showing the C source code, but AFAICT we're actually showing the GOTO?